### DISCUSSION TITLE
Custom tagging and filtering..

### DISCUSSION THREAD

**Discussion by jvrobert** [blakeblackshear/frigate#2910] [Ideas]
Maybe this has been discussed, couldn't find anything but it would be pretty cool if e.g. double-take (or some external process) could tag people in frigate events, which could then be filtered in the events GUI.

So e.g. Frigate detects person, double-take (or anything else) gets event or queries API, finds "Joe" in image. It then calls back into frigate API with image name, bounding box, and frigate stores it. Then from UI you can do a "custom tag" dropdown and filter based on "Joe".

Is something like this feasible? Seems like easy way to integrate facial recognition directly without all the heavy lifting.

**Comment by NickM-27** [comment]
I actually just made an issue the other day for this: https://github.com/blakeblackshear/frigate/issues/2900

I am waiting on Blake's input for that but I am planning on implementing it myself assuming Blake agrees 👍
